src/Core/src/Credentials/Credentials.php

```diff
@@ -62,4 +68,22 @@ public function getCredentials(Configuration $configuration): ?Credentials
     {
         return $this->isExpired() ? null : $this;
     }
+
+    public static function adjustExpireDate(?\DateTimeImmutable $expireDate, ?\DateTimeImmutable $reference = null): ?\DateTimeImmutable
+    {
+        if (null === $expireDate) {
+            return $expireDate;
```

Comment:
I missed this. Could we change this?
```suggestion
            return null;
```

Reply:
can be deleted since `adjustExpireDate` expect a `\DateTimeImmutable`